firmware/Reload Pro.cydsn/ui.c

```diff
@@ -108,13 +112,24 @@ const menudata choose_readout_menu = {
 	}
 };
 
+const menudata tools_menu_data = {

```

Comment:
Following the convention of existing menu names, this should just be `tools_menu`.